config/env/local_client-external.json

```diff
@@ -4,18 +4,21 @@
       "mode": "client",
       "apiURL": "http://localhost:7007"
     },
-    "ceramic2": {
-      "mode": "client",
-      "apiURL": "https://ceramic-dev.3boxlabs.com",
-      "anchorServiceAPI": "https://cas-dev.3boxlabs.com",
-      "ethereumRpc": null,
-      "network": "dev-unstable"
-    },
     "ipfs": {
       "mode": "none"
     },
+    "ceramic2": {
+      "mode": "client",
+      "apiURL": "https://ceramic-dev.3boxlabs.com"
+    },
     "ipfs2": {
       "mode": "none"
     }
-  }
+  },
+  "serviceLauncher": [
```

Comment:
Cool, I do think this looks clearer.  Not sure about the name though.  Having `serviceLauncher` be an array of services doesn't seem quite right.  Maybe the two top-level objects should be `jestLaunchedServices` and `externallyLauncedServices`?  Or `services` and `preLaunchedServices`?